### ISSUE TITLE
Skip records screened layer

### ISSUE THREAD

**Issue by chriscpritchard** [prisma-flowdiagram/PRISMA2020#71]
It would be nice to be able to skip the "Records screened" layer. Sometimes we proceed directly to full-text retrieval and eligibility appraisal. (taken from #70)

**Comment by KevinKlipsch** [comment]
Agree. Also, it would be nice to have more flexibility in general as to which box is shown and which not. I find the PRISMA flowchart very nice, but I am not doing full systematic reviews in a medical context. Instead, literature searches on a specific topic, which are then screenen at abstract level. I would only need one level at the screening stage like: Records/abstracts screened (n=...) and on the right a box with the exclusion reasons. Probably a big ask, but ideally there would be full flexibiltiy to remove or add complete levels (like Identification, Screening, Included, possibly more) and which and how many boxes to include at each level...
